.github/workflows/nightly-release.yml

```diff
@@ -0,0 +1,123 @@
+                })
+            } catch (e) {
+              console.log("Warning: The nightly tag doesn't exist yet, so there's nothing to do. Trace: " + e)
+            }
+            await github.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: "refs/tags/nightly",
+              sha: context.sha
+            })
+      # This will create a new GitHub Release called `nightly`
+      # If a release with this name already exists, it will overwrite the existing data
+      - name: Create a nightly GitHub prerelease
+        id: create_prerelease
+        uses: ncipollo/release-action@v1
+        with:
+          name: nightly
+          artifacts: "${{ env.BUILD_OUTPUT_LIST }}"
+          tag: nightly
+          bodyFile: ".github/workflows/nightly-release-readme.md"
+          prerelease: true
+          allowUpdates: true
+          removeArtifacts: true
+          draft: false
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish nightly GitHub prerelease
+        uses: eregon/publish-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          release_id: ${{ steps.create_prerelease.outputs.id }}
+  # Send a slack notification if either job defined above fails
+  slack-notify:
+    needs:
+      - build-artifacts
+      - github-release
+    if: always() && (needs.build-artifacts.result == 'failure' || needs.github-release.result == 'failure')
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Send slack notification on failure
+        uses: slackapi/slack-github-action@v1.18.0
+        with:
+          payload: |
+            {
+              "text": ":x::moon::nomad-sob: Damon Nightly Release *FAILED*",
+              "attachments": [
+                {
+                  "color": "#C41E3A",
+                  "blocks": [
+                    {
+                      "type": "section",
+                      "fields": [
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Branch:*\n`${{ github.ref_name }}`"
+                        },
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Ref:*\n${{ github.sha }}"
+                        }
+                      ]
+                    },
+                    {
+                      "type": "section",
+                      "fields": [
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Workflow:*\n${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+                        }
+                      ]
+                    }
+                  ]
+                }
+              ]
+            }
+        env:
+          # the slack webhook url links to #feed-nomad-releases
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
@hcjulz This secret also needs to be set in the repo config. We can use the same bot we have for the other Nomad repos, or you can use your own if you have it already.